src/main/java/plugily/projects/murdermystery/arena/Arena.java

```diff
@@ -80,7 +80,8 @@
   private final List<Stand> stands = new ArrayList<>();
   private final List<SpecialBlock> specialBlocks = new ArrayList<>();
   private final List<Player> allMurderer = new ArrayList<>(), allDetectives = new ArrayList<>();
-
+  private final List<Player> spectators = new ArrayList<>();
+  private final List<Player> deaths = new ArrayList<>();
```

Comment:
```suggestion
  private final List<Player> allMurderer = new ArrayList<>(), allDetectives = new ArrayList<>(), spectators = new ArrayList<>(), deaths = new ArrayList<>();

```
Keep cleaned-up the code